### PR TITLE
Reference code fixed. (IDFGH-9066)

### PR DESCRIPTION
**Pull request by gasaimaple**
Fixed a small code bug in reference document.

PS: I have no idea why this goes wrong when the english version is correct.

### CODE REVIEW

**Comment by Alvin1Zhang** [comment]
Thanks for your contribution.

File: docs/zh_CN/api-reference/protocols/esp_http_server.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@ HTTP Server 组件提供了在 ESP32 上运行轻量级 Web 服务器的功能
         esp_err_t get_handler(httpd_req_t *req)
         {
             /* 发送回简单的响应数据包 */
-            const char[] resp = "URI GET Response";
+            const char resp[] = "URI GET Response";
```
Comment (suda-morris) [inline docs/zh_CN/api-reference/protocols/esp_http_server.rst:25]: Would you please fix Line58 as well, it has the same issue.

**Comment by mahavirj** [comment]
> PS: I have no idea why this goes wrong when the english version is correct.

It was fixed in https://github.com/espressif/esp-idf/pull/3940, but seems that we missed out on propagating the fix to Chinese version of the docs.

**Comment by hmalpani** [comment]
SHA=27fc115b98127094790067c82abb0a3237b9f97a

**Comment by mahavirj** [comment]
Merged with https://github.com/espressif/esp-idf/commit/ed1b2ba60e033c58131d4d2e2f554533a5e1527b

**Pull request closed without merging**